target/linux/ath79/dts/qca9531_glinet_gl-ar300m-lite.dts

```diff
@@ -1,3 +1,5 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
 /dts-v1/;
 
 #include "qca9531_glinet_gl-ar300m-nor.dts"
```

Comment:
You have one DTS including another one here. This should be changed so that both DTS files use a common DTSI.

Reply:
> From my understanding, base-files is shared between subtargets (like generic or nand)

The build system is a bit more sophisticated than that. It will include from both the PLATFORM_DIR and the PLATFORM_SUBDIR (see `include/target.mk`, as an example). 

This effectively allows common files to be placed in the PLATFORM_DIR, with overrides and additions in the PLATFORM_SUBDIR. 

So, for example, "generic", "tiny", and "nand" all would have the content in `ath79/base-files` in their ROMs, but the content from `ath79/nand/base-files`, _which is meaningless for "tiny" devices,_ as well, overriding that in the platform-level directory.

So yes, the primary reasons for using `nand/base-files/`include:
* NAND-based devices and their code do not impact the installed weight of "generic" and "tiny" targets.
* Easier maintenance going forward by isolating class-specific scripts and information to those devices where it is applicable

I can't speak as to why the ramips directory is arranged as it is. Or. for that matter why the "tiny" targets didn't adopt this approach. As I've seen in your PRs, there are a lot of things form the past that could be done in a more uniform and maintainable fashion. 

---

That would then create `qca9531_glinet_gl-ar300m-nor.dts` which would  consist of only 
```
/dts-v1/;

 #include "qca9531_glinet_gl-ar300m-nor.dtsi"
```
which seems pretty wonky, just to maintain a naming convention. Then again, on this, I perhaps can see a future use case where it is of value.

Reply:
The inclusion of the DTS for the GL-AR300M-Lite  is currently on `master` and `openwrt-19.07`. I've addressed that with https://patchwork.ozlabs.org/patch/1170891/

I'll adjust here for the devices that are the subject this PR.

Reply:
Resolved by https://patchwork.ozlabs.org/patch/1170891/